--- gmudix-1.0/src/gui_user.c
+++ gmudix-1.0/src/gui_user.c	2009-05-12 22:36:38.000000000 +0200
@@ -820,7 +820,7 @@
 
     dialog = gtk_message_dialog_new(GTK_WINDOW(user->gui_user.g_window), 
                 GTK_DIALOG_DESTROY_WITH_PARENT, GTK_MESSAGE_INFO, 
-                GTK_BUTTONS_CLOSE, msg);
+                GTK_BUTTONS_CLOSE, '%s', msg);
                 
     gtk_window_set_transient_for(GTK_WINDOW(user->gui_user.g_window), 
                                  GTK_WINDOW(dialog));
